packages/renderer/src/lib/pod/PodDetails.spec.ts

```diff
@@ -51,6 +52,7 @@ const removePodMock = vi.fn();
 
 beforeAll(() => {
   (window as any).listPods = listPodsMock;
+  (window as any).listContainers = listContainersMock.mockResolvedValue([]);
   (window as any).kubernetesListPods = kubernetesListPodsMock;
   (window as any).removePod = removePodMock;
 });
```

Comment:
we're missing the test of checking if we have links to open URLs and verify that if we click on the button we're redirected to the expected link